HISTORY.rst

```diff
@@ -6,6 +6,9 @@ History
 development
 ++++++++++++++++++
 
+Features:
+
+* Refactor MozillaOIDCAuthenticationBackend to support more OIDC servers
```

Comment:
The first part is true "Refactor MozillaOIDCAuthenticationBackend" but just because some of the big methods are broken up doesn't make it support more OIDC servers. Perhaps a sentence like "Refactor MozillaOIDCAuthenticationBackend so that the algorithm and verification key can be overridden in subclasses when you need to ..."

Reply:
done